packages/actor-abstract-path/test/PathVariableObjectIterator-test.ts

```diff
@@ -10,7 +10,7 @@ import { Factory } from 'sparqlalgebrajs';
 import { PathVariableObjectIterator } from '../lib';
 
 const DF = new DataFactory();
-const BF = new BindingsFactory();
+const BF = new BindingsFactory({});
```

Comment:
Could we make the merge handlers optional in the constructor?
Then we don't need changes in places like this.